daemon/api.go

```diff
@@ -143,7 +143,7 @@ var (
 	snapstateRevertToRevision  = snapstate.RevertToRevision
 	snapstateSwitch            = snapstate.Switch
 
-	assertstateRefreshSnapDeclarations = assertstate.RefreshSnapDeclarations
```

Comment:
I might be wrong, but it seems to me that after this change `assertstate.RefreshSnapDeclarations` is unused. Should we remove it?

Reply:
It is used a lot in some targeted tests (which could maybe get updated to just call RefreshAssertions), but I'd like to know what @pedronis thinks about this PR before going too far.